fe1-web/src/features/evoting/objects/__tests__/ElectionPrivateKey.test.ts

```diff
@@ -0,0 +1,56 @@
+import { curve } from '@dedis/kyber';
+import Ed25519Scalar from '@dedis/kyber/curve/edwards25519/scalar';
+
+import { Base64UrlData } from 'core/objects';
+import { mockEncodedElectionPrivateKey } from 'features/evoting/__tests__/utils';
+
+import { ElectionPrivateKey } from '../ElectionPrivateKey';
+
+const ed25519 = curve.newCurve('edwards25519');
+
+describe('ElectionPrivateKey', () => {
+  describe('constructor', () => {
+    it('does not throw an error', () => {
+      expect(() => new ElectionPrivateKey(mockEncodedElectionPrivateKey)).not.toThrow();
+      expect(() => new ElectionPrivateKey(mockEncodedElectionPrivateKey)).not.toThrow();
+    });
+  });
+
+  describe('toString()', () => {
+    it('does not throw an error', () => {
+      const key = new ElectionPrivateKey(mockEncodedElectionPrivateKey);
+      expect(() => key.toString()).not.toThrow();
+    });
+  });
+
+  describe('toBase64()', () => {
+    it('does not throw an error', () => {
+      const key = new ElectionPrivateKey(mockEncodedElectionPrivateKey);
+      expect(() => key.toBase64()).not.toThrow();
+    });
+  });
```

Comment:
Wouldn't it be better to check that the type conversion worked for both these tests?

Reply:
Yes you are totally right!